MAS/MAS.py

```diff
@@ -832,8 +832,20 @@ def Analyse_RowAs_Kmeans(self):
         print("mean_nb_RALs", means_grp)
         
         if (p_value < 0.0001):
+
+            # Find the group with the minimum mean
+            min_mean_group_index = np.argmin(means_grp)
+
+            _indeces_grp0 = np.array(_indeces_grp0)
+            _indeces_grp1 = np.array(_indeces_grp1)
+
+            # Get the indices associated with the group with the minimum mean
+            index_small_grp = list((_indeces_grp0, _indeces_grp1))[min_mean_group_index]
+
+            _indeces_grp0 = tuple(_indeces_grp0)
+            _indeces_grp1 = tuple(_indeces_grp1)
```

Comment:
I don't think these conversions are necessary. 🤔 Can you explain why you did them?